actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1479,6 +1479,9 @@ def time_field(object_name, method, options = {})
       # The default value is generated by trying to call +strftime+ with "%Y-%m-%dT%T"
       # on the object's value, which makes it behave as expected for instances
       # of DateTime and ActiveSupport::TimeWithZone.
+      # If you pass "include_seconds: false", it will be
+      # formatted by trying to call +strftime+ with "%Y-%m-%dT%H:%M" on the object's value.
+      # It is also possible to override this by passing the "value" option.
```

Comment:
This "The default value is generated by..." paragraph is a prelude for the example directly below it.  Instead of appending new information here (which isn't demonstrated in the example), what about adding another paragraph and example further below, similar to [what #41728 did](https://github.com/rails/rails/pull/41728/files#diff-44b94eca66c7497711821a8e6bcdfde4684bb7b8efa15e64da6532449f03ef0bR1428-R1433)?